<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentDialog</class>
 <widget class="QDialog" name="StudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: rgb(255, 255, 255);
}

QLineEdit{
	border:1px solid gray;
	border-radius:6px;
	padding-left:15px;
}

QDateEdit{
	border:1px solid gray;
    border-radius:6px;
	padding-left:15px;
}</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>167</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(90, 90, 90);</string>
   </property>
   <property name="text">
    <string>Date Of Birth</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>167</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(90, 90, 90);</string>
   </property>
   <property name="text">
    <string>Class</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>513</width>
     <height>67</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(90, 90, 90);</string>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="address_lineEdit">
      <property name="minimumSize">
       <size>
        <width>511</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>511</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>513</width>
     <height>67</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(90, 90, 90);</string>
      </property>
      <property name="text">
       <string>Phone Number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="phone_lineEdit">
      <property name="minimumSize">
       <size>
        <width>511</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>511</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>167</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(90, 90, 90);</string>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>513</width>
     <height>67</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(90, 90, 90);</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="email_lineEdit">
      <property name="minimumSize">
       <size>
        <width>511</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>511</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveStudent_btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>520</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#34D481;
color:white;
border:none;
border-radius:8px;
font-weight:bold;
font-size:15px;</string>
   </property>
   <property name="text">
    <string>Add Student</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>520</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #585858;
color:white;
border:none;
border-radius:8px;
font-weight:bold;
font-size:15px;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins Medium</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(44, 44, 44);</string>
   </property>
   <property name="text">
    <string>Add New Student</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>513</width>
     <height>67</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(90, 90, 90);</string>
      </property>
      <property name="text">
       <string>Full Names</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="name_lineEdit">
      <property name="minimumSize">
       <size>
        <width>511</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>511</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>511</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <widget class="QComboBox" name="gender_comboBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"> QComboBox {
                border: 2px solid #ffffff;
                border-radius: 8px;
                padding: 1px 18px 1px 3px;
                background: black;
                selection-background-color: #2980b9;
				padding-left:15px;
                color: white;
				font-weight:bold;
            }</string>
      </property>
      <item>
       <property name="text">
        <string>Select Gender</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="class_comboBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"> QComboBox {
                border: 2px solid #ffffff;
                border-radius: 8px;
                padding: 1px 18px 1px 3px;
                background: black;
                selection-background-color: #2980b9;
				padding-left:15px;
                color: white;
				font-weight:bold;
            }</string>
      </property>
      <item>
       <property name="text">
        <string>Select Class</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grade 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grade 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grade 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grade 4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grade 5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grade 6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grade 7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grade 8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grade 9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grade 10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grade 11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grade 12</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dob_dateEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
